mex-wholebodymodel/matlab/wrappers/+WBM/+utilities/eul2rotm.m

```diff
@@ -0,0 +1,64 @@
+function rotm = eul2rotm(eul, sequence)
```

Comment:
I did not copied them from somewhere. I have written them by my self. In a lot of these files I have added a comment from where I have the sources. I.e. the literature, papers, or when I have used parts of code as a template from a foreign package, I added there also the links of them.